### PR TITLE
add examples of making an object publicly-readable

### PR DESCRIPTION
**Pull request by stephenplusplus**
RE: #350

### CODE REVIEW

File: lib/storage/index.js [hunk 1]
```diff
@@ -131,6 +131,18 @@ function Storage(config) {
  * var storage = gcloud.storage();
  * var albums = storage.bucket('albums');
  *
+ * //-
+ * // Make a bucket's contents publicly readable.
```
Comment (ryanseys) [inline lib/storage/index.js]: Can you elaborate here to say that new items added to the bucket will be made public by default?

Comment (stephenplusplus) [inline lib/storage/index.js]: Good catch. I think this snippet would only apply to the objects already in the bucket. Maybe I should use `albums.acl.default.add` instead?

Comment (ryanseys) [inline lib/storage/index.js]: Oops, good catch on my good catch! I didn't actually realize they were different. You should definitely clarify whether it makes new objects public or not which I think is the standard case here. The use case that I'm imagining is a website asset storage bucket.

**Comment by stephenplusplus** [comment]
@ryanseys added a new example. PTAL!

**Pull request merged**